projects/ngx-json-schema-form/src/lib/layout-node.ts

```diff
@@ -54,6 +54,10 @@ export class LayoutNode {
             ? pointer
             : `/${pointer.replace(/\./g, '/')}`;
     }
+    /** Copy of the original layoutItem */
+    @Memoize() get layoutDefinition(): LayoutItem {
```

Comment:
While this works, should also consider using layoutservice to fetch layoutitem from map.  that way layoutitem is not immediately accessible from layoutnode

Reply:
Will consider this in the future, if it proves helpful/necessary